client/app/iplb/sslCertificate/order/iplb-ssl-certificate-order.controller.js

```diff
@@ -67,8 +67,9 @@ class IpLoadBalancerSslCertificateOrderCtrl {
     }
 
     orderFreeCertificate () {
-        const fqdn = this.newSsl.fqdn.split(",");
-
+        const fqdn = this.newSsl.fqdn.split(",").map(function(item){
```

Comment:
Hello, that's nice ! You can use arrow function following to ES6 code. In that case : 
`const fqdn = this.newSsl.fqdn.split(",").map(item => item.trim());`